src/components/shared/AccountMenu/AccountMenu.tsx

```diff
@@ -137,8 +137,7 @@ const AccountMenu: FC<AccountMenuProps> = ({user}) => {
                                         </ListItemButton>
                                     </ListItem>
                                     <ListItem disablePadding>
-                                        <ListItemButton
-                                            onClick={() => handleClick("https://forms.gle/Ue9wQDXDuczLk7b56")}>
+                                        <ListItemButton onClick={() => handleClick("https://forms.gle/Ue9wQDXDuczLk7b56")}>
```

Comment:
Will this open on a new tab? Probably better for accessibility as well if we use an `a` or any wrapper component thereof.